### ISSUE TITLE
lib/util/logger.py

### ISSUE THREAD

**Issue by dservais** [notadamking/RLTrader#134]
The logger.py in lib/util has multiple issues in the lines

fh = logging.FileHandler(os.path.join('data', log', 'trading.warning.log')
to be replaced by
fh = logging.FileHandler(os.path.join('data', 'log', 'trading.warning.log'))

**Comment by LeaveMyYard** [comment]
Yeah, seems like he didn't found this error cause of this part of code being commented
